### PR TITLE
Enabled backlogged transfers

### PR DESCRIPTION
**Pull request by Hwesta**
Added backlog purpose to location and config for default location.  POSTing to a Location with a pipeline, origin location, and list of files will move the files from the provided location to the location posted to.  Fixed some problems with moving folders.

Supports AM PR for SIP Arrangement.

### CODE REVIEW

File: storage_service/storage_service/urls.py [hunk 1]
```diff
@@ -58,6 +58,11 @@ def startup():
         space=space,
         relative_path=os.path.join('var', 'archivematica', 'sharedDirectory', 'www', 'AIPsStore'),
         description='Store AIP in standard Archivematica Directory')
+    backlog, _ = locations_models.Location.objects.get_or_create(
+        purpose=locations_models.Location.BACKLOG,
+        space=space,
+        relative_path=os.path.join('var', 'archivematica', 'sharedDirectory', 'www', 'AIPsStore', 'transferBacklog'),
```
Comment (mistydemeo) [inline storage_service/storage_service/urls.py:64]: The `relative_path` is meant to be relative to `/`?

**Comment by mistydemeo** [comment]
:thumbsup:

File: storage_service/storage_service/urls.py (hunk 1 above)
Comment (Hwesta) [inline storage_service/storage_service/urls.py:64]: Because the default Space (created line 45) is '/'.

Comment (mistydemeo) [inline storage_service/storage_service/urls.py:64]: Ah, right.

**Pull request merged**